pipeline_documentation.md

```diff
@@ -1016,25 +1016,25 @@ Filter multimappers favoring mismatches over InDels with a
 - **Examples**
 
 ```console
-Example 1 | Different proportion of mismatches vs. InDels
+Example 1 | Different amount of InDels
```

Comment:
Different _number_

Reply:
It would be good to specifiy somewhere the number of mismatches and InDels in each alignment, as it's difficult to parse the CIGAR and MD strings together. Same applies in the example below.

Reply:
And/or include the actual alignments (see other comment in script below) so that it becomes immediately obvious.